aten/src/ATen/core/typeid.h

```diff
@@ -17,18 +17,19 @@
 
 #include <exception>
 
-#include "ATen/core/Error.h"
 #include "ATen/core/Backtrace.h"
-#include "ATen/core/Macros.h"
+#include "ATen/core/C++17.h"
+#include "ATen/core/Error.h"
 #include "ATen/core/Half.h"
 #include "ATen/core/IdWrapper.h"
+#include "ATen/core/Macros.h"
```

Comment:
So, I was expecting to see a comment in this file saying how the new scheme worked (since it can be somewhat difficult to puzzle out exactly what the macros are doing), but I didn't see anything like that. Can we have a comment like that? Now I have to go read the macros and figure out what the new ones do.

Reply:
I added some comments